src/engine/renderer/tr_image.cpp

```diff
@@ -1819,11 +1819,20 @@ image_t *R_FindImageFile( const char *imageName, imageParams_t &imageParams )
 			}
 
 			// Built-in images can't be reloaded with different parameters, so return them as-is.
-			// For most of the usable ones e.g. _white, parameters wouldn't make a difference anyway.
+			// For most of the usable ones e.g. $white, parameters wouldn't make a difference anyway.
 			// HACK: detect built-in images by naming convention, though nothing stops users from using such names
-			if ( image->name[ 0 ] == '_' && !strchr( image->name, '/' ) )
+			switch ( image->name[ 0 ] )
 			{
-				return image;
+				case '_': // TODO: Remove once _fog is removed.
```

Comment:
You can remove `_` now since it's not an image anyone needs to load from a shader.

Reply:
Done. Also this `_fog` image is now NUKED anyway.